dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -76,6 +79,9 @@ def profile(self):
             profile["statistics"].update(
                 dict(categories=self.categories)
             )
+            profile["statistics"]['categorical_count'] = dict(
+                sorted(self._categories.items(), key=itemgetter(1),
+                       reverse=True)[:num_top_categories])
```

Comment:
just to confirm this is descending? e.g. largest number is on top? put a comment mentioning that maybe?